### ISSUE TITLE
update owncloud-server-scl-php54-8.1.0-16.1.noarch on CentOS 6

### ISSUE THREAD

**Issue by jessegilbert** [owncloud/core#18503]
Hi there -

I have owncloud installed on a CentOS 6 server, which has been working fine for a while.  I currently have version 8.1.0-16.1 installed via the SCL PHP54 package, to get around the PHP 5.4 requirement.  When trying to update via Yum I'm getting dependency errors:

# yum update
Loaded plugins: product-id, rhnplugin, security, subscription-manager
Setting up Update Process
Resolving Dependencies
--> Running transaction check
---> Package owncloud-3rdparty.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-3rdparty = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-3rdparty.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-activity.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-activity = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-activity.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-encryption.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-encryption = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-encryption.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-external.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-external = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-external.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-files.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-files = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-files.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-files_external.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-files_external = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-files_external.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-files_locking.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-files_locking = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-files_locking.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-files_pdfviewer.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-files_pdfviewer = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-files_pdfviewer.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-files_sharing.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-files_sharing = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-files_sharing.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-files_texteditor.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-files_texteditor = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-files_texteditor.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-files_trashbin.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-files_trashbin = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-files_trashbin.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-files_versions.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-files_versions = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-files_versions.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-files_videoviewer.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-files_videoviewer = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-files_videoviewer.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-firstrunwizard.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-firstrunwizard = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-firstrunwizard.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-gallery.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-gallery = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-gallery.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-provisioning_api.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-provisioning_api = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-provisioning_api.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-templateeditor.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-templateeditor = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-templateeditor.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-user_external.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-user_external = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-user_external.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-user_ldap.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-user_ldap = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-user_ldap.noarch 0:8.1.1-12.1 will be an update
---> Package owncloud-app-user_webdavauth.noarch 0:8.1.0-16.1 will be updated
--> Processing Dependency: owncloud-app-user_webdavauth = 8.1.0 for package: owncloud-server-scl-php54-8.1.0-16.1.noarch
---> Package owncloud-app-user_webdavauth.noarch 0:8.1.1-12.1 will be an update
--> Finished Dependency Resolution
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-3rdparty = 8.1.0
           Removing: owncloud-3rdparty-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-3rdparty = 8.1.0-16.1
           Updated By: owncloud-3rdparty-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-3rdparty = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-user_webdavauth = 8.1.0
           Removing: owncloud-app-user_webdavauth-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-user_webdavauth = 8.1.0-16.1
           Updated By: owncloud-app-user_webdavauth-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-user_webdavauth = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-files_sharing = 8.1.0
           Removing: owncloud-app-files_sharing-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-files_sharing = 8.1.0-16.1
           Updated By: owncloud-app-files_sharing-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-files_sharing = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-firstrunwizard = 8.1.0
           Removing: owncloud-app-firstrunwizard-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-firstrunwizard = 8.1.0-16.1
           Updated By: owncloud-app-firstrunwizard-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-firstrunwizard = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-external = 8.1.0
           Removing: owncloud-app-external-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-external = 8.1.0-16.1
           Updated By: owncloud-app-external-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-external = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-files_trashbin = 8.1.0
           Removing: owncloud-app-files_trashbin-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-files_trashbin = 8.1.0-16.1
           Updated By: owncloud-app-files_trashbin-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-files_trashbin = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-files_pdfviewer = 8.1.0
           Removing: owncloud-app-files_pdfviewer-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-files_pdfviewer = 8.1.0-16.1
           Updated By: owncloud-app-files_pdfviewer-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-files_pdfviewer = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-files_texteditor = 8.1.0
           Removing: owncloud-app-files_texteditor-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-files_texteditor = 8.1.0-16.1
           Updated By: owncloud-app-files_texteditor-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-files_texteditor = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-encryption = 8.1.0
           Removing: owncloud-app-encryption-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-encryption = 8.1.0-16.1
           Updated By: owncloud-app-encryption-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-encryption = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-activity = 8.1.0
           Removing: owncloud-app-activity-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-activity = 8.1.0-16.1
           Updated By: owncloud-app-activity-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-activity = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-user_external = 8.1.0
           Removing: owncloud-app-user_external-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-user_external = 8.1.0-16.1
           Updated By: owncloud-app-user_external-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-user_external = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-provisioning_api = 8.1.0
           Removing: owncloud-app-provisioning_api-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-provisioning_api = 8.1.0-16.1
           Updated By: owncloud-app-provisioning_api-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-provisioning_api = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-files_versions = 8.1.0
           Removing: owncloud-app-files_versions-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-files_versions = 8.1.0-16.1
           Updated By: owncloud-app-files_versions-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-files_versions = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-templateeditor = 8.1.0
           Removing: owncloud-app-templateeditor-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-templateeditor = 8.1.0-16.1
           Updated By: owncloud-app-templateeditor-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-templateeditor = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-files_external = 8.1.0
           Removing: owncloud-app-files_external-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-files_external = 8.1.0-16.1
           Updated By: owncloud-app-files_external-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-files_external = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-files = 8.1.0
           Removing: owncloud-app-files-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-files = 8.1.0-16.1
           Updated By: owncloud-app-files-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-files = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-files_locking = 8.1.0
           Removing: owncloud-app-files_locking-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-files_locking = 8.1.0-16.1
           Updated By: owncloud-app-files_locking-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-files_locking = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-files_videoviewer = 8.1.0
           Removing: owncloud-app-files_videoviewer-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-files_videoviewer = 8.1.0-16.1
           Updated By: owncloud-app-files_videoviewer-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-files_videoviewer = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-gallery = 8.1.0
           Removing: owncloud-app-gallery-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-gallery = 8.1.0-16.1
           Updated By: owncloud-app-gallery-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-gallery = 8.1.1-12.1
Error: Package: owncloud-server-scl-php54-8.1.0-16.1.noarch (@isv_ownCloud_community)
           Requires: owncloud-app-user_ldap = 8.1.0
           Removing: owncloud-app-user_ldap-8.1.0-16.1.noarch (@isv_ownCloud_community)
               owncloud-app-user_ldap = 8.1.0-16.1
           Updated By: owncloud-app-user_ldap-8.1.1-12.1.noarch (isv_ownCloud_community)
               owncloud-app-user_ldap = 8.1.1-12.1

It appears the owncloud-server-scl-php54 needs an update, but I can't seem to find one.  Any thoughts?

**Comment by karlitschek** [comment]
@jnweiger any idea?

**Comment by jnweiger** [comment]
This might have been related to or even a duplicate of https://github.com/owncloud/core/issues/21462
Fixed today. Please reopen if there is more.
